src/components/popover/examples/popover-in-dialog.tsx

```diff
@@ -0,0 +1,101 @@
+import { Option } from '@limetech/lime-elements';
+import { Component, h, State } from '@stencil/core';
+
+/**
+ * This example shows how a popover works when used within a dialog.
+ * Nested popups are possible using the dialog, popover and select components
+ * together but it's not recommended use for a good user experience.
+ */
+@Component({
+    tag: 'limel-example-popover-dialog',
+    shadow: true,
+    styleUrl: 'popover.scss',
+})
+export class PopoverInDialogExample {
+    @State()
+    private isDialogOpen = false;
+
+    @State()
+    private isPopoverOpen = false;
+
+    @State()
+    public value: Option;
+
+    private options: Option[] = [
+        { text: 'Luke Skywalker', value: 'luke' },
+        { text: 'Han Solo', value: 'han', disabled: true },
+        { text: 'Leia Organo', value: 'leia' },
+    ];
+
+    constructor() {
+        this.openPopover = this.openPopover.bind(this);
+        this.onPopoverClose = this.onPopoverClose.bind(this);
+        this.openDialog = this.openDialog.bind(this);
+        this.onDialogClose = this.onDialogClose.bind(this);
+        this.onSelectChange = this.onSelectChange.bind(this);
+    }
+
+    public render() {
+        return [
+            <limel-button
+                primary={true}
+                label="Open dialog"
+                onClick={this.openDialog}
+            />,
+            <limel-dialog
+                heading="Selected hero"
+                open={this.isDialogOpen}
+                onClose={this.onDialogClose}
+            >
+                <limel-flex-container>
+                    {this.value?.text || 'None'}
+                    <limel-popover
+                        open={this.isPopoverOpen}
+                        onClose={this.onPopoverClose}
+                    >
+                        <limel-icon-button
+                            slot="trigger"
+                            label="Change"
+                            icon="pencil"
+                            onClick={this.openPopover}
+                        />
+                        <limel-select
+                            value={this.value}
+                            options={this.options}
+                            onChange={this.onSelectChange}
+                            style={{ width: '30vw' }}
+                        />
+                    </limel-popover>
+                </limel-flex-container>
```

Comment:
I think this example feels a bit too artificial to really be great, and it's also not working great, at least not on my machine (when opening the select, the page scrolls up to the top, but the menu stays where it was, making it "disappear" on my 13 inch laptop screen; I'm not sure if it's a problem with the example or with one of our components though). I prefer any example to be something we'd like people to copy and use in production code.

Reply:
@adrianschmidt That sounds weird! Yeah, I agree that the example is a bit unrealistic and I don't think we really need an example to prove/show how the z-index works either. I'll remove that so we can merge this baby.